.travis.yml

```diff
@@ -15,7 +15,7 @@ branches:
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
 
 env:
-  - BAZEL=0.13.0 TF=NIGHTLY
+  - BAZEL=0.16.0 TF=NIGHTLY
```

Comment:
Can we add a line to the code that uses 

https://github.com/bazelbuild/bazel-skylib/blob/daf513702286fe211f291675443235e35e79f34f/lib/versions.bzl#L82

```BUILD
load("@bazel_skylib//lib/versions.bzl", "versions")
versions.check(minimum_bazel_version = "0.16.0")
```